.github/workflows/actions.yml

```diff
@@ -75,6 +77,7 @@ jobs:
       run: |
         pip install -r requirements.txt
         pip install -e ".[tests]" --progress-bar off --upgrade
+        pip install keras-nlp-nightly
```

Comment:
Shouldn't we also use `--no-deps` here? or is there a reason to be different these actions?

Reply:
for keras 3 tests, it can pull all the nightly versions of tf-text and tf-nightly -  it is okay. but with keras 2 tests the tf versions clash and results in error